scripts/dist-git/garbage_collection

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Cleanup unnecessary files
+
+REPODIR=$(crudini --get dist-git.conf dist-git gitroot_dir)
+cd $REPODIR
+
+collect()
+{
+    if [ "$1" -gt "1" ]; then
```

Comment:
Can you document what is the semantics of $1 and why the service script calls it with 2 by default? comment at the top of the file is fine.